python-setup/python-setup-suggestions.qmd

```diff
@@ -0,0 +1,154 @@
+
+So how do you do it? To create a new environment:
+
+1. Open the Command Palette with **Command + Shift + P** (Mac) or **Control + Shift + P** (Windows and Linux)
+
+![Command Palette](images/palette.png)
+
+2. Select the **Python: Create Environment** command.
+
+3. Then select the type of environment you would like to create: Venv or Conda.
+
+![Select type](images/type.png)
+
+4. Next select the Python interpreter (venv) or Python version (conda) that you would like to use for the environment.
+
+![Select interpreter](images/interpreter.png)
+
+Positron will create the environment in you current folder and then activate it for use. Along the way, it will do several things:
+
+- It will install any package dependencies listed in a `requirements.txt`, `pyproject.toml`, or `environment.yml` file that appears in your folder. This is helpful if you've cloned your project from a git collaborator and wish to quickly recreate their environment.
+- It will add your environment directory to your folder's `.gitignore` file, so you will be less likely to commit your environment to github. More on that below.
+
+Now that you have an environment, you can equip it for your needs by installing packages into it.
+
+## Install packages
+
+Each virtual environment in Python starts with a minimal set of packages that includes:
+
+- The Python Standard Library 
+- pip
+- setuptools
+- wheels
+
+As we saw above, Positron will also install any packages listed in a `requirements.txt`, `pyproject.toml`, or `environment.yml` file that appears in your directory.
+
+But what if you need to install additional packages? You can do this from the Terminal Window with your preferred method. Many Python users will choose to use Python's `pip` module. To use it:
+
+1. Activate your Python environment in the terminal with the command below. This code assumes that your environment is saved in a folder named `.venv`.
+
+```bash
+source .venv/bin/activate
+```
+
+2. Use the pip install command to install the packages you desire, e.g.,
+
+```bash
+pip install shiny
+```
+
+![Using pip to install the shiny package](images/pip.png)
+
+Python will install the packages into your currently active environment.
+
+Now that you have your environment setup, it is time to document it for your future self and collaborators. 
+
+## Document your environment
+
+Reproducing your Python data science project requires two to three things:
+
+1. Your data (likely)
+1. Your code 
+1. Your environment
+
+But it would be bad manners to check in your environment to your version control system. For one, your environment directory can be quite large. For two, checking in your environment is unnecessary: you can easily document everything one would need to recreate the environment for themselves.
+
+Let's look at two popular ways to do that.
+
+### requirements.txt
```

Comment:
Again, I think I would show _one_ fairly standard way and link off to other possibilities like poetry, uv, etc.

Reply:
Done. https://github.com/garrettgman/positron-guides/pull/1/commits/bbd62a337aed3a758ef790db99b6712c62247015